src/Windows/Avalonia.Direct2D1/PrimitiveExtensions.cs

```diff
@@ -127,13 +127,17 @@ public static StrokeStyle ToDirect2DStrokeStyle(this Avalonia.Media.Pen pen, Fac
                 EndCap = pen.EndLineCap.ToDirect2D(),
                 DashCap = pen.DashCap.ToDirect2D()
             };
-            var dashes = new float[0];
+            float[] dashes = null;//If we new an array first that we may waste it when we use `dashes = pen.DashStyle.Dashes.Select(x => (float)x).ToArray()`. As a frequently used code I think this optimize is necessary.
```

Comment:
Could you remove this comment too? We really don't need a comment explaining what this is doing, it's pretty obvious ;)

Reply:
@grokys Ok.